### DISCUSSION TITLE
Build watch paths as file

### DISCUSSION THREAD

**Discussion by ari-party** [coollabsio/coolify#2680] [Feature Requests]
![image](https://github.com/coollabsio/coolify/assets/49074962/8b884bee-4b12-44a4-bfb5-0715391fb848)
Watch paths as a file, e.g. `.coolifywatch`
Includes itself ofc

**Comment by aelmiger** [comment]
I am on the newest version 306 and I dont see the options of Watch Paths in the config. Instead, it only says: Custom Docker Options.
Where do I find/activate the Watch Paths option to begin with?

**Reply by ari-party** [reply]
![Screenshot_20240703-124528.png](https://github.com/coollabsio/coolify/assets/49074962/e251cf11-8f38-4bf6-a10a-aa1bea7022a3)

Available on Nixpacks and Dockerfile, don't know about the rest

**Reply by aelmiger** [reply]
Thanks for the quick response.
Strange! I am using nixpacks, but this option is missing.
